homeassistant/helpers/update_coordinator.py

```diff
@@ -237,7 +237,7 @@ async def _async_refresh(  # noqa: C901
             self._unsub_refresh()
             self._unsub_refresh = None
 
-        self._debounced_refresh.async_cancel()
+        self._debounced_refresh.async_cancel(keep_cooldown=True)
```

Comment:
I am not sure if this is the best way to resolve.
It seems that `UpdateCoordinator` is banking on the cooldown timer being there regardless.

Maybe it should be a new `pause` method instead of `cancel`